examples/speech_recognition/modules/specaugment.py

```diff
@@ -0,0 +1,79 @@
+"""SpecAugment Implementation for Pytorch.
+Related paper : https://arxiv.org/pdf/1904.08779.pdf
+"""
+
+import torch
+import torch.nn as nn
+import numpy as np
+import random
+
+
+class SpecAugment(nn.Module):
```

Comment:
may you please add some docs to this class and its methods? Thanks.

Reply:
Updated in the last commit